doc/telescope.txt

```diff
@@ -1118,6 +1197,56 @@ action_state.get_current_picker({prompt_bufnr})*action_state.get_current_picker(
 
 
 
+================================================================================
+                                                             *telescope.resolve*
+
+Provides "resolver functions" to allow more customisable inputs for options.
+
+resolver.resolve_height()                          *resolver.resolve_height()*
+    Converts input to a function that returns the height. The input must take
+    one of four forms:
+    1. 0 <= number < 1
+       This means total height as a percentage.
+    2. 1 <= number
+       This means total height as a fixed number.
+    3. function
+       Must have signature: function(self, max_columns, max_lines): number
+    4. table of the form: {padding = `foo`} <br> where `foo` has one of the
+       previous three forms. <br> The height is then set to be the remaining
+       space after padding. For example, if the window has height 50, and the
+       input is {padding = 5}, the height returned will be `40 = 50 - 2*5`
+
+    The returned function will have signature: function(self, max_columns,
+    max_lines): number
+
+
+
+resolver.resolve_width()                            *resolver.resolve_width()*
+    Converts input to a function that returns the width. The input must take
+    one of four forms:
+    1. 0 <= number < 1
+       This means total width as a percentage.
+    2. 1 <= number
+       This means total width as a fixed number.
+    3. function
+       Must have signature: function(self, max_columns, max_lines): number
+    4. table of the form: {padding = `foo`} <br> where `foo` has one of the
+       previous three forms. <br> The width is then set to be the remaining
+       space after padding. For example, if the window has width 100, and the
+       input is {padding = 5}, the width returned will be `90 = 100 - 2*5`
+
+    The returned function will have signature: function(self, max_columns,
+    max_lines): number
+
+
+
+resolver.win_option()                                  *resolver.win_option()*
+    Win option always returns a table with preview, results, and prompt. It
+    handles many different ways. Some examples are as follows:
```

Comment:
Here is a lot more documentation in the source file. Not really important for the enduser so we could either make the two lines into only 2 `-` or we make the other 19 lines into 3 `-` (with some `<br>`s) up to you

Reply:
I agree that this is not as important for the end user, so would be fine to get rid if we want.